docs/react/use-with-create-react-app-ts.en-US.md

```diff
@@ -0,0 +1,236 @@
+---
+order: 4
+title: Use in create-react-app-ts
+---
+
+[create-react-app](https://github.com/facebookincubator/create-react-app) is one of best React application development tool, we are going to use `antd` within it and modify the webpack config for some customized needs.
+
+---
+
+## Install and Initialization
+
+We need to install `create-react-app` first, you may need install [yarn](https://github.com/yarnpkg/yarn/) too.
+
+Additionally, install [react-scripts-ts](https://github.com/wmonk/create-react-app-typescript)
```

Comment:
No need install this, right ?
all you need is specify the scripts version.
```sh
--scripts-version=react-scripts-ts 
```

Reply:
you're right